doc/ref_model/refs.bib

```diff
@@ -348,3 +348,587 @@ @misc{anuket-abbreviatons
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63b.pdf}
+}
+
+@misc{NIST-SP-800-115,
+	title = {{NIST SP 800-115 Technical Guide to Information Security Testing and Assessment}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-115.pdf}
+}
+
+@misc{NIST-SP-800-125,
+	title = {{NIST SP 800-125 Guide to Security for Full Virtualization Technologies}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-125.pdf}
+}
+
+@misc{NIST-SP-800-125a,
+	title = {{NIST SP 800-125a Security Recommendations for Server-based Hypervisor Platforms}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-125Ar1.pdf}
+}
+
+@misc{NIST-SP-800-125b,
+	title = {{NIST SP 800-125b Secure Virtual Network Configuration for Virtual Machine (VM) Protection}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-125B.pdf}
+}
+
+@misc{NIST-SP-800-137,
+	title = {{NIST SP 800-137 Information Security Continuous Monitoring for Federal Information Systems and Organizations}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-137.pdf}
+}
+
+@misc{NIST-SP-800-145,
+	title = {{NIST SP 800-145 The NIST Definition of Cloud Computing}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-145.pdf}
+}
+
+@misc{NIST-SP-800-190,
+	title = {{NIST SP 800-190 Application Container Security Guide}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-190.pdf}
+}
+
+@misc{NIST-SP-800-145,
+	title = {{The NIST Definition of Cloud Computing}},
+	url = {https://csrc.nist.gov/pubs/sp/800/145/final}
+}
+
+@misc{TS-103-457cyber,
+	title = {{CYBER;
+	Trusted Cross-Domain Interface:
+	Interface to offload sensitive functions to a trusted domain}},
+	url = {https://www.etsi.org/deliver/etsi_ts/103400_103499/103457/01.01.01_60/ts_103457v010101p.pdf}
+}
+
+@misc{jenkins,
+	title = {{Jenkins}},
+	url = {https://www.jenkins.io/}
+}
+
+@misc{x,
+	title = {{y}},
+	url = {https://docs.gitlab.com/ee/ci/}
+}
+
+@misc{ci/cd-gitlab,
+	title = {{Gitlab CI/CD}},
+	url = {z}
+}
+
+@misc{xtestingCI,
+	title = {{XtestingCI}},
+	url = {https://galaxy.ansible.com/collivier/xtesting}
+}
+
+@misc{CIS-password-policy,
+	title = {{CIS Password Policy Guide}},
+	url = {https://www.cisecurity.org/insights/white-papers/cis-password-policy-guide}
+}
+
+@misc{the-18-CIS-critical-security-controls,
+	title = {{The 18 CIS Critical Security Controls}},
+	url = {https://www.cisecurity.org/controls/cis-controls-list}
+}
+
+@misc{CVE-program-mission,
+	title = {{CVE® Program Mission}},
+	url = {https://cve.mitre.org}
+}
+
+@misc{common-vulnerability-scoring,
+	title = {{Common Vulnerability Scoring System SIG}},
+	url = {https://www.first.org/cvss/}
+}
+
+@misc{center-internet-security,
+	title = {{Center for Internet Security}},
+	url = {https://www.cisecurity.org/}
+}
+
+@misc{cloud-security-alliance,
```

Comment:
This is a duplicate entry.

Reply:
Removed the @misc{cloud-security-alliance,